src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1413,6 +1413,21 @@
   <data name="UseAsSpanInsteadOfRangeIndexerOnAnArrayCodeFixTitle" xml:space="preserve">
     <value>Use `{0}` instead of Range-based indexers on an array</value>
   </data>
+  <data name="PlatformCompatabilityCheckTitle" xml:space="preserve">
+    <value>Call of platform dependent API</value>
```

Comment:
Need more descriptive resource strings I think 🤔. Can we get more inputs for it?

Reply:
By my understanding for the title, the shorter is better, no? cc @terrajobst @jeffhandley

Reply:
@mavasani where does the title show up?